pkg/controller/datavolume/smart-clone-controller.go

```diff
@@ -369,22 +370,32 @@ func newPvcFromSnapshot(name string, snapshot *snapshotv1.VolumeSnapshot, target
 		common.CDILabelKey:       common.CDILabelValue,
 		common.CDIComponentLabel: common.SmartClonerCDILabel,
 	}
+
```

Comment:
This is a massive nitpick but I'd drop this empty line, as done with annotations below